bundles/org.openhab.binding.knx/src/main/resources/OH-INF/i18n/knx.properties

```diff
@@ -107,45 +102,50 @@ channel-type.knx.switch.description = A channel to manage a generic Group Addres
 channel-type.config.knx.dimmer.position.label = Address
-channel-type.config.knx.dimmer.position.description = The group address(es) in Group Address Notation to set the absolute position of the dimmer
+channel-type.config.knx.dimmer.position.description = The group address to set the absolute brightness (see format description above).
 channel-type.config.knx.dimmer.switch.label = Address
-channel-type.config.knx.dimmer.switch.description = The group address(es) in Group Address Notation to toggle the dimmer on or off
+channel-type.config.knx.dimmer.switch.description = The group address (GA) to toggle the dimmer on or off. Additional listening (status) GAs can be added by concatenating GAs with a "+" (e.g. "1/7/18+2/9/15"). The DPT can be defined by prepending it to the first GA, separated by ":" (e.g. "2.001:1/4/17"). GAs prepended with a "<" (e.g. "<5/1/125") are read from the bus on startup.
 channel-type.config.knx.rollershutter.position.label = Address
-channel-type.config.knx.rollershutter.position.description = The group address(es) in Group Address Notation to set the absolute position of the shutter, in %
+channel-type.config.knx.rollershutter.position.description = The group address to set the absolute position of the shutter, in % (see format description above).
 channel-type.config.knx.rollershutter.stopMove.label = Address
-channel-type.config.knx.rollershutter.stopMove.description = The group address(es) in Group Address Notation to start (MOVE) or STOP shutter movement
+channel-type.config.knx.rollershutter.stopMove.description = The group address to start (MOVE) or STOP shutter movement (see format description above).
 channel-type.config.knx.rollershutter.upDown.label = Address
-channel-type.config.knx.rollershutter.upDown.description = The group address(es) in Group Address Notation to move the shutter in the DOWN or UP direction
+channel-type.config.knx.rollershutter.upDown.description = The group address (GA) to set the moving direction (up/down). Additional listening (status) GAs can be added by concatenating GAs with a "+" (e.g. "1/7/18+2/9/15"). The DPT can be defined by prepending it to the first GA, separated by ":" (e.g. "2.001:1/4/17"). GAs prepended with a "<" (e.g. "<5/1/125") are read from the bus on startup.
 channel-type.config.knx.single.ga.label = Address
-channel-type.config.knx.single.ga.description = The group address(es) in Group Address Notation
+channel-type.config.knx.single.ga.description = The group address (GA) for this channel. Additional listening (status) GAs can be added by concatenating GAs with a "+" (e.g. "1/7/18+2/9/15"). The DPT can be defined by prepending it to the first GA, separated by ":" (e.g. "2.001:1/4/17"). GAs prepended with a "<" (e.g. "<5/1/125") are read from the bus on startup.
+
+# add-on
+
+addon.knx.name = KNX Binding
+addon.knx.description = This binding supports connecting to a KNX bus
```

Comment:
Moved unintentionally? Usually this is in the beginning of the file.

Reply:
This is done by the i18n tooling when I regenerated the translations. I gave up to move it back to the beginning. Have you seen this in other binding as well or could it be related to my setup?

Reply:
For the main branch, for me running `mvn i18n:generate-default-translations` gives this result:
![image](https://user-images.githubusercontent.com/19519842/226184285-6c46078c-bde0-478c-a14b-8053be8c60f0.png)

Reply:
@jlaur thank you for checking... I just had a look at my batch file... and found the "3.4.0" I had overlooked before. Now it works fine.